hyperspy/learn/mva.py

```diff
@@ -958,6 +959,170 @@ def plot_explained_variance_ratio(self, n=50, log=True):
             ax.semilogy()
         return ax
 
+    def scree_plot(self, n=50, log=True, cutoff=0.01, signal_num=0,
+                   xaxis=None, signal_fmt=None, noise_fmt=None,
+                   axes_titles=None, fig=None, ax=None,
+                   **kwargs):
+        """Plot the decomposition explained variance ratio vs index number
+        (Scree Plot). This method is an alternative to
+        `plot_explained_variance_ratio`
+
+        Parameters
+        ----------
+        n : int
+            Number of components to plot.
+        log : bool
+            If True, the y axis uses a log scale.
+        cutoff : float or None
+            y-position at which to draw a line showing the cutoff between
+            signal and noise
+        signal_num : int
+            Number of components to highlight as important
+            (i.e. the signal components as opposed to noise).
+            Only used if `cutoff` is None, will be automatically determined
+            otherwise
+        xaxis : dict or None
+            Dictionary defining type of labeling for x-axis. Should
+            have two keys: ``'type'`` and ``'labeling'``.
+            ``xaxis['type']`` should have str value ``'index'`` or
+            ``'number'``, and is used to determine if axis will be labeled
+            starting at 0 (i.e. "pythonic index" labeling) or at 1 (number
+            labeling).
+            ``xaxis['labeling']`` should have str value ``'ordinal'`` or
+            ``'cardinal'``, and is used to determine the format of the tick
+            labels (i.e. "1st" or "1", respectively).
+        signal_fmt : dict
+            Dictionary of matplotlib formatting values for the signal
+            components
+        noise_fmt : dict
+            Dictionary of matplotlib formatting values for the noise
+            components
+        axes_titles: dict or None
```

Comment:
It is very easy to use mpl to customize the axes labels, why providing another way?

From the zen of Python

> There should be one-- and preferably only one --obvious way to do it.